client/scripts/helpers/snapshot_utils.ts

```diff
@@ -183,74 +185,96 @@ export async function getProposals(space: string): Promise<SnapshotProposal[]> {
+  const web3 = new Web3Provider((window as any).ethereum);
+  const receipt = await client.vote(web3 as any, address, payload);
+}
+
+export async function createProposal(address: string, payload: any) {
+  const web3 = new Web3Provider((window as any).ethereum);
+
+  const receipt = await client.proposal(web3 as any, address, payload);
+  return receipt;
+}
+
 export async function getSpaceBlockNumber(network: string): Promise<number> {
-  const { default: Snapshot } = await import('@snapshot-labs/snapshot.js');
-  return Snapshot.utils.getBlockNumber(Snapshot.utils.getProvider(network));
+  return snapshot.utils.getBlockNumber(snapshot.utils.getProvider(network));
 }
 
 export async function getScore(space: SnapshotSpace, address: string) {
-  const { default: Snapshot } = await import('@snapshot-labs/snapshot.js');
-  return Snapshot.utils.getScores(
+  return snapshot.utils.getScores(
     space.id,
     space.strategies,
     space.network,
-    [address],
+    [address]
     // Snapshot.utils.getProvider(space.network),
   );
 }
 
 /* Single Choice Voting */
 
-export async function getResults(space: SnapshotSpace, proposal: SnapshotProposal) {
-  const { default: Snapshot } = await import('@snapshot-labs/snapshot.js');
+export async function getResults(
+  space: SnapshotSpace,
+  proposal: SnapshotProposal
+) {
+  console.log(space, proposal);
   try {
     let votes = await getVotes(proposal.id);
-
-    // const voters = votes.map(vote => vote.voter);
-    // const provider = Snapshot.utils.getProvider(space.network);
     const strategies = proposal.strategies ?? space.strategies;
+
     if (proposal.state !== 'pending') {
-      const scores = await Snapshot.utils.getScores(
-        space.id,
-        strategies,
-        space.network,
-        votes.map((vote) => vote.voter),
-        parseInt(proposal.snapshot, 10),
-        // provider,
-      );
-      votes = votes.map((vote: any) => {
-          vote.scores = strategies.map(
-            (strategy, i) => scores[i][vote.voter] || 0
-          );
-          vote.balance = vote.scores.reduce((a, b: any) => a + b, 0);
-          return vote;
-        })
-        .sort((a, b) => b.balance - a.balance)
-        .filter(vote => vote.balance > 0);
+      try {
+        const scores = await snapshot.utils.getScores(
+          space.id,
+          strategies,
+          space.network,
+          votes.map((vote) => vote.voter),
+          parseInt(proposal.snapshot, 10)
+          // provider,
+        );
+        votes = votes
+          .map((vote: any) => {
+            vote.scores = strategies.map(
+              (strategy, i) => scores[i][vote.voter] || 0
+            );
+            vote.balance = vote.scores.reduce((a, b: any) => a + b, 0);
+            return vote;
+          })
+          .sort((a, b) => b.balance - a.balance)
+          .filter((vote) => vote.balance > 0);
+      } catch (e) {
+        console.log(e);
```

Comment:
we should let the user know that it failed, or try a few more times (bc snapshot api is fickle)